### PR TITLE
Add chrome in Travis ("Next Up" No.2 of TODO.md)

### PR DESCRIPTION
**Pull request by RustyNail**
Hi, @luke-hill.
I have implemented the following tasks of TODO list.

```
2) Travis:
  - Add in Chrome
  - Update Driver/Browser
```

## Change point
- .travis.yml
  - Added chrome and its driver.
  - Update geckodriver to latest version.
- features/support/env.rb
  - Make profile according to browser.
- TODO.md
  - Deleted items implemented by this Pullrequest.

### Ref
- Setting up Google Chrome with travis
  - https://docs.travis-ci.com/user/chrome
- Latest geckodriver
  - https://github.com/mozilla/geckodriver/releases/

### CODE REVIEW

File: features/support/env.rb [hunk 1]
```diff
@@ -44,10 +44,15 @@
 end
 
 Capybara.register_driver :selenium do |app|
-  profile = Selenium::WebDriver::Firefox::Profile.new
+  browser = ENV['TARGET_BROWSER'].to_sym || :firefox
```
Comment (luke-hill) [inline features/support/env.rb]: Could we maybe put this in a private method along with `browser == :chrome` -> Probably call the second method `chrome?`

Comment (RustyNail) [reply features/support/env.rb]: @luke-hill , Thank you to review.

I changed it, is it such an image? > https://github.com/natritmeyer/site_prism/pull/214/commits/fe9afdda11841a75873eec8408ced035756dab6b

**Comment by luke-hill** [comment]
I'll cherry-pick some of your commits and rebuild the PR sometime on Friday.

**Comment by luke-hill** [comment]
Closing in favour of #217 - Which enhances this functionality slightly with fewer commits.

**Comment by RustyNail** [comment]
Thank you for improvement. @luke-hill 
Supporting local execution environments is good.
I learned a lot.

I would like to continue contributing to solving some issues in "TODO.md".

**Comment by luke-hill** [comment]
That's great to here. Thanks again for your help. This will hit `2.12` You will be credited as the primary author, with myself as a secondary author

**Pull request closed without merging**